documentation/src/main/asciidoc/topics/con_xsite_auto_offline.adoc

```diff
@@ -0,0 +1,53 @@
+[id='xsite_auto_offline-{context}']
+= Taking Sites Offline Automatically
+Backup configurations include timeout values for operations to replicate
+data to remote sites. When backup operations reach the timeout, {brandname}
+records the operation as a failure.
+
+To automatically take sites offline, you can then configure the
+number of **consecutive** failures that can occur.
+
+For example, the **NYC** backup configuration specifies five as the number of
+failures after which **NYC** goes offline. If **LON** attempts five consecutive
+backup operations that fail, {brandname} automatically takes **NYC** offline.
+**LON** then stops backing up to **NYC** until you bring the site back online.
+
+[source,xml,options="nowrap",subs=attributes+]
+----
+<backup site="NYC" strategy="ASYNC">
+	<take-offline after-failures="5"/>
+</backup>
+----
+
+You can also specify a minimum wait time for backup operations to complete
+before sites go offline, for example:
+
+[source,xml,options="nowrap",subs=attributes+]
+----
+<backup site="NYC" strategy="ASYNC">
+	<take-offline after-failures="5"
+                min-wait="10000"/>
+</backup>
+----
+
+In the preceding example, **NYC** goes offline if backup operations take longer than 10 seconds and failures occur for 5 consecutive operations.
```

Comment:
not `if backup operations take longer than 10 seconds` but rather there is no successful request in these 10 seconds.

Reply:
also fixed. thanks again.